index.html

```diff
@@ -14,17 +14,18 @@
       padding: 0;
       font-family: sans-serif;
       background-color: #ededed;
+      height: 100%;
     }
   </style>
   <title>sidekick-library</title>
 </head>
 
 <body>
-  <script type="module" src="./src/app.js"></script>
+  <script type="module" src="./src/index.js"></script>
   <script>
     const library = document.createElement('sidekick-library');
     library.config = {
-      base: 'https://main--helix-test-content-onedrive--adobe.hlx.page/block-library-tests/library-single-sheet.json'
+      base: 'https://main--boilerplate-with-library--dylandepass.hlx.page/tools/sidekick/library.json',
```

Comment:
shouldn't we move this to a public location?

Reply:
Hey @tripodsan, this html file is only used for development and is not deployed to `hlx.live`. Customer will created their own `library.html` file which will be equivalent to this one.